g2p/transducer/__init__.py

```diff
@@ -522,7 +525,11 @@ def apply_unidecode(self, to_convert: str):
         tg = TransductionGraph(to_convert)
 
         # Conversion is done character by character using unidecode
-        converted = [text_unidecode.unidecode(c) for c in to_convert]
+        converted = [unicodedata.normalize("NFKC", c) for c in to_convert]
```

Comment:
maybe change to `[text_unidecode.unidecode(unicodedata.normalize("NFKC", c)) for c in to_convert]` just to save the extra loop

Reply:
I don't think it's equivalent. I had the same thought, but NFKC can expand a char in several chars, then you loop again after expansion.

Reply:
You could do it in nested loops, but this is clearer I think.

Reply:
`converted` will still be a list of strings though, regardless of whether `unicodedata.normalize("NFKC", c)` returns a different length string. So, I think they should actually be equivalent.

Reply:
Sorry, that was nonsense, since we're working with arrays, we're not joining in between.

So yes, the three loops could be converted to a single, equivalent, loop, it's true.

Reply:
no, just the first 2, since the 3rd has a conditional dependent on the output of the first two transformations.